demos/sev-demo/README.md

```diff
@@ -0,0 +1,206 @@
+# CCv0 with AMD SEV
+
+CCv0 can be used with AMD SEV. Check out this [demo video](https://youtu.be/SXZ-hzhjahU) to see it in action. This guide explains how to take an SEV node and use CCv0 with it. This guide does not cover configuring an SEV machine. You need to have a node where you can start SEV VMs. This guide also does not cover setting up CCv0. You need a working deployment of CCv0. You will update this deployment to use SEV. 
+Soon many of these steps will be automated via [Kata Deploy](https://github.com/kata-containers/kata-containers/tree/main/tools/packaging/kata-deploy) and the [Confidential Containers Operator](https://github.com/confidential-containers/operator).
+
+## OVMF
+
+You will need to use OVMF. Furthemore, you will need to use the AMDSEV build of OVMF. An appropriate [firmware binary](OVMF.fd) is available in this directory.
+The binary in this directory is built from commit `f0f3f5aae7c4d346ea5e24970936d80dc5b60657` and has md5 checksum `bc40de8b6e743e03b3912de64e7d4884`. Any release after [edk2-stable202108](https://github.com/tianocore/edk2/releases/tag/edk2-stable202108) should work. 
+
+You can also build from source. First clone the source and checkout the submodules.
```

Comment:
Let's make it clear that building an OVMF is optional and the recommended approach is to use the included binary (because it's easier)